### DISCUSSION TITLE
Hydration Error

### DISCUSSION THREAD

**Discussion by FelixJncs** [neuefische/questions#602] [Web] [answered]
Hey, i got some Error messages and do not know how to get rid of them. 

1. This Error occurs immediately: 
![Screenshot 2023-05-12 at 10 29 29](https://github.com/neuefische/questions/assets/125086329/707c4973-0f5b-4c18-8cc2-65da3ead9804)

It hints that there is an a tag in another a tag. I was not able to find the tags and do not see any other Errors. 

[My Project](https://github.com/FelixJncs/capstone-project/tree/User_Story_3) I was not able to detect the Problem, some assistance would be much appreciated. 

Thanks in advance😀

**Comment by marcbendixen** [comment, marked as answer]
Hey @FelixJncs 👋 
one problem could be this: you are using `a`-tags inside the `Link` component without the `legacyBehavior` prop.

Example `index.js`
```js
<Link href="/addhabit">
    <StyledAddLink>Add New Habit</StyledAddLink>
</Link>
```
Should look like this:
```js
<Link href="/addhabit" legacyBehavior>
    <StyledAddLink>Add New Habit</StyledAddLink>
</Link>
```

You can find this in the docs [here](https://nextjs.org/docs/app/api-reference/components/link#if-the-child-is-a-tag). This needs to be updated everywhere you are using it.
